packages/authorization-process/test/getProducerKeychainUsers.test.ts

```diff
@@ -0,0 +1,64 @@
+import { getMockProducerKeychain } from "pagopa-interop-commons-test/src/testUtils.js";
+import {
+  ProducerKeychain,
+  ProducerKeychainId,
+  TenantId,
+  UserId,
+  generateId,
+} from "pagopa-interop-models";
+import { describe, expect, it } from "vitest";
+import { genericLogger } from "pagopa-interop-commons";
+import {
+  organizationNotAllowedOnProducerKeychain,
+  producerKeychainNotFound,
+} from "../src/model/domain/errors.js";
+import { addOneProducerKeychain, authorizationService } from "./utils.js";
+
+describe("getProducerKeychainUsers", async () => {
+  const organizationId: TenantId = generateId();
+  const userId1: UserId = generateId();
+  const userId2: UserId = generateId();
+  const mockProducerKeychain: ProducerKeychain = {
+    ...getMockProducerKeychain(),
+    users: [userId1, userId2],
+    producerId: organizationId,
+  };
```

Comment:
Let's avoid sharing data between tests, I would declare these inside the tests where are necessary (I think only in the first one, for the rest you could just use `getMockProducerKeychain`)